Release foxglove-schemas-{flatbuffer,protobuf} 0.4.0 (#1416)
foxglove/foxglove-sdk: CONTRIBUTING.md

2. Manually create a new Release in the GitHub UI. Ensure the tag uses the form `typescript/schemas/vX.Y.Z`.
3. GitHub Actions will take care of the rest.

### Python schema packages (`foxglove-schemas-protobuf`, `foxglove-schemas-flatbuffer`)

> These packages are versioned and released independently from the SDK. Generated bindings are produced at build time (not committed); use the Docker-backed targets so `protoc` / `flatc` match CI:

Reviewer: ```suggestion
These packages are versioned and released independently from the SDK. Generated code is produced at build time.

To build and run tests using Docker:
```

```sh
make build-python-schemas
make test-python-schemas
```

> 1. Create and merge a PR bumping the `version` in the package's `pyproject.toml` (and the corresponding entry in its `uv.lock`).

Reviewer: ```suggestion
To make a release:

1. Create and merge a PR bumping the `version` in the package's `pyproject.toml` (and the corresponding entry in its `uv.lock`).
```

> 2. From the merged commit on `main`, push a tag of the form:

Reviewer: ```suggestion
2. From the merged commit on `main`, push a tag (or create a [Release](https://github.com/foxglove/foxglove-sdk/releases)) of the form:
```

   - `python/foxglove-schemas-protobuf/vX.Y.Z`, or
   - `python/foxglove-schemas-flatbuffer/vX.Y.Z`

> 3. The `schemas` job in `.github/workflows/python.yml` will build the package and publish it to PyPI when it sees the matching tag.

Reviewer: ```suggestion
3. The `schemas` job in `.github/workflows/python.yml` will automatically build the package and publish it to PyPI when the tag is pushed.
```
